container/sagemaker/tensorflow-serving.js

```diff
@@ -13,9 +13,68 @@ function invocations(r) {
     }
 }
 
-// ping check does not POST to a specific model, because MME starts with 0 model
+function mme_ping(r) {
+    // hack for MME
+    // MME starts without any model loaded and the default-tfs-model is set to "None"
+    var uri = make_tfs_uri(r, true)
+    var options = {
+        method: 'POST',
+        body: '{"instances": "invalid"}'
+    }
+
+    function callback (reply) {
+        if (reply.status == 200 || reply.responseBody.includes('"Servable not found for request: Latest(None)"')) {
```

Comment:
can we just check the status code here, or is that not specific enough?

Reply:
The status code is 502, I do not think it's specific enough to tell the case.